app/views/todo_lists/_card.html.slim

```diff
@@ -1,11 +1,9 @@
 .card.todo_list_card class="bg-#{todo_list_card.color_theme} #{todo_list_card.light? ? '' : 'text-white'}"
-  span.badge.badge-light = todo_list_card.tags
+  / span.badge.badge-light = todo_list_card.tags
   h3.card-header.text-center = todo_list_card.name
   .card-body
-    p.card-text = todo_list_card.description
   ul.list-group.list-group-flush
     - todo_list_card.todo_items.take(2).each do |item|
-      // Boostrap override le bg-color pour les items, on pourrait décider de le changer en css custom aussi
```

Comment:
mmh ouai why not a voir je vois pas le truc la faudra que tu m'explique de vive voix